cumulusci/salesforce_api/tests/test_package_install.py

```diff
@@ -183,4 +185,5 @@ def test_package_install_options_from_task_options():
         name_conflict_resolution=NameConflictResolution.RENAME,
         password="foo",
         security_type=SecurityType.PUSH,
+        apex_compile_type=ApexCompileType.PACKAGE,
     )
```

Comment:
Include upgrade_type in the test too?

Reply:
I deliberately left it out to validate that there wasn't any negative behavior with its nullability.

Reply:
Perhaps I should make that test explicit.

Reply:
Ah okay. Wouldn't hurt to have both tests but we don't need to hold this up.